pkg/actions/templates.go

```diff
@@ -80,7 +81,7 @@ func AddTemplateRepo(c *cli.Context) {
 	}
 
 	conID := strings.TrimSpace(strings.ToLower(c.String("conid")))
-	repos, err := apiroutes.AddTemplateRepo(conID, url, desc, name, gitCredentials)
+	repos, err := templates.AddTemplateRepo(conID, url, desc, name, gitCredentials)
```

Comment:
yes I wasn't sure how to cut up our code to avoid huge files.

The previous logic flow was `pkg/actions/templates.go` -> `pkg/apiroutes/templates.go`, and `apiroutes` just handled communication with PFE. 

But now we need more than just communicating with PFE, because e.g. we need to save and extract git credentials from the keychain. So I thought we could use a `pkg/templates/templates.go` to coordinate template stuff.

So I changed the flow to `pkg/actions/templates.go` -> `pkg/templates/templates.go` -> `pkg/apiroutes/templates.go`.

But I'm open to ideas. I notice that our `project` package handles communication with PFE too, rather than calling a `pkg/apiroutes/projects.go` file